CHANGES.md

```diff
@@ -1,3 +1,6 @@
+# 2.19.0 (2019-10-07)
+- [IMPROVED] Add getter for total row count to AllDocsResponse
```

Comment:
Please change to
```
# Unreleased
- [NEW] Add getter for total row count to AllDocsResponse
```

We'll fill in the date and version when we cut the release and I think this should be a `NEW` because it is adding API. Thanks!

Reply:
@ricellis I've pushed that changes. Thank you for the explanation of the correct pattern to use for pagination, this topic was causing some confusion for us.